specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5784,7 +5784,25 @@
             "Replace",
             "Merge",
             "Delete"
-          ]
+          ],
+          "x-ms-enum": {
+            "name": "operation",
+            "modelAsString": false,
```

Comment:
How about set to `true` for adding new value in the future?

Reply:
Sure, @raych1 could you please shed a bit more light on this **'modelAsString'**? What is this for and what is the difference for setting it true or false? I can change it to 'true' in my pr if needed. I just did something similar to the property **PropertyChangeType**.

Reply:
It enable to add more values for `enum` type in the future when set `modelAsString` as true. It won't be breaking change. Pls refer [x-ms-enum](http://azure.github.io/autorest/extensions/#x-ms-enum) for details.

Reply:
Hello @raych1, thanks for the explanation. I have set it to true. Since @ravbhatnagar mentioned that ARM review board would not be reviewing property-level changes going forward. Could you please have another look and merge it?